ansible/roles/seldon/defaults/main.yaml

```diff
@@ -8,9 +8,83 @@ seldon_install_runtime: true
 seldon_install_servers: true
 
 seldon_force_install_crds: false
+seldon_force_install_components: false
+seldon_force_install_runtime: false
+seldon_force_install_servers: false
 seldon_core_v2_timeout_helm: "480s"
-save_helm_components_overrides: false
-save_helm_components_overrides_file: "{{ ansible_env.HOME }}/seldon_helm_comp_values.yaml"
 
 seldon_core_v2_component_values: {}
 seldon_core_v2_servers_values: {}
+
+seldon_default_dev:
+  dockerhub_user: "seldonio"
+  install_kind_images: true
+  skip_build_images: false
+  save_helm_components_overrides: false
+  save_helm_components_overrides_file: "{{ ansible_env.HOME }}/seldon_helm_comp_values.yaml"
+
+seldon_default_image_config:
+  registry: "{{ omit }}"
```

Comment:
Does `{{ omit }}` carry any special meaning?

Reply:
It acts as the key hasn't been set (in this case, the registry key will not appear in the helm chart config unless overwritten by the extra vars file). As opposed to it being set to an empty value.